### ISSUE TITLE
No package 'gladeui-2.0' found

### ISSUE THREAD

**Issue by jonyosn** [rhinstaller/anaconda#1322]
in execute ./configure ,
![image](https://user-images.githubusercontent.com/19390690/35952841-e80c158e-0cbc-11e8-9290-04aa4b2afe4b.png)

**Comment by jkonecny12** [comment]
Hello, this is totally correct. You are missing those packages and you need to install them before you can run the configure successfully.

If you don't want to pick those packages one by one a look for them and you are trying to build master branch of Anaconda then you can use script which will return you all the required dependencies. This script is located in ``./scripts/testing/dependency_solver.py`` .

**Comment by jonyosn** [comment]
when i execute "yum install -y $(./scripts/testing/dependency_solver.py)",and my repos include epel.
![image](https://user-images.githubusercontent.com/19390690/35970174-5ea5f722-0d04-11e8-8100-0f678d8a4ee3.png)

**Comment by jkonecny12** [comment]
Yes, you have RHEL 7 and this script is designed for Fedora 27 and newer.

You can still run it inside of mock by:

```
./scripts/testing/setup-mock-test-env.py fedora-27-x86_64 --init --copy
mock -r fedora-27-x86_64 --shell
<inside of mock>
cd /anaconda && ./autogen && ./configure && make
```

**Comment by poncovka** [comment]
Thank you for reporting this issue! The issue tracker will be closed on 3 June 2019. Please make sure that your issue is also reported at https://bugzilla.redhat.com/. We are handling a high number of issues and many of them involve other components. Bugzilla allows us to better process your issues and collaborate on them with other teams. We are sorry for the inconvenience.
